### PR TITLE
Fix GPU init action

### PR DESCRIPTION
**Pull request by medb**

### CODE REVIEW

**Comment by functicons** [comment]
Could you check why the presubmit is failing.

File: gpu/install_gpu_driver.sh [hunk 1]
```diff
@@ -14,124 +14,118 @@
...
-
-    # Add non-free Debian 9 Stretch packages.
-    # See https://www.debian.org/distrib/packages#note
-    for type in deb deb-src; do
-      for distro in stretch stretch-backports; do
-        for component in contrib non-free; do
-          echo "${type} http://deb.debian.org/debian/ ${distro} ${component}" \
-              >> /etc/apt/sources.list.d/non-free.list
-        done
+function install_gpu() {
```
Comment (functicons) [inline gpu/install_gpu_driver.sh]: s/install_gpu/install_gpu_driver

File: gpu/install_gpu_driver.sh [hunk 2]
```diff
@@ -14,124 +14,118 @@
...
 # This script installs NVIDIA GPU drivers and collects GPU utilization metrics.
 
-set -exo pipefail
+set -euxo pipefail
 
 # Download settings
 readonly REQUIREMENTS_URL='https://raw.githubusercontent.com/GoogleCloudPlatform/ml-on-gcp/master/dlvm/gcp-gpu-utilization-metrics/requirements.txt'
 readonly REPORT_GPU_URL='https://raw.githubusercontent.com/GoogleCloudPlatform/ml-on-gcp/master/dlvm/gcp-gpu-utilization-metrics/report_gpu_metrics.py'
 
+INSTALL_GPU_AGENT="$(/usr/share/google/get_metadata_value attributes/install_gpu_agent || true)"
```
Comment (functicons) [inline gpu/install_gpu_driver.sh]: Add comment about what GPU Agent is.

**Comment by medb** [comment]
> Could you check why the presubmit is failing.

Presubmit failing because of not enough GPU quota in CI project, it succeeds when run locally.

**Pull request merged**